### ISSUE TITLE
Fullscreen mode for non-fullscreen canvas

### ISSUE THREAD

**Issue by Fischaela** [borismus/webvr-boilerplate#149]
Hey all, 

I am having a canvas that is not a fullscreen canvas. When I use the fullscreen button, the whole browser page goes into fullscreen mode, but the canvas stays non-fullscreen. 

Is this an intended behavior? Wouldn't it be cool, if the canvas would go full screen, too – e.g. if it would transform into a canvas with `width: 100vw;` and `height: 100vh;` before going into the actual fullscreen mode? 

Cheers, Michaela

**Comment by devasur** [comment]
May be I am missing something here.  Wouldn't it be something that can easily be controlled with CSS? how can the boiler plate decide which canvas on the document to transform?

**Comment by borismus** [comment]
Right, boilerplate currently fullscreens the whole page. This is largely to
include the VR transitioning UI that it provides (which is rendered on top
of the canvas, using DOM)
